rasa/core/policies/ted_policy.py

```diff
@@ -182,12 +187,9 @@ class TEDPolicy(Policy):
 
     @staticmethod
     def _standard_featurizer(max_history: Optional[int] = None) -> TrackerFeaturizer:
-        if max_history is None:
-            return FullDialogueTrackerFeaturizer(LabelTokenizerSingleStateFeaturizer())
-        else:
-            return MaxHistoryTrackerFeaturizer(
-                LabelTokenizerSingleStateFeaturizer(), max_history=max_history
-            )
+        return MaxHistoryTrackerFeaturizer(
```

Comment:
@evgeniiaraz Could you please explain this change?

Reply:
basically: why are they needed?

Reply:
do you mean, why we got rid of "if max_history" statement? because with the current  changes is `MaxHistoryTrackerFeaturizer` it can handle any length (up to the beginning) and, from what I've heard from Vova, `MaxHistoryTrackerFeaturizer` always worked better `FullDialogueTrackerFeaturizer`.